.claude/docs/pr-testing-agent.md

```
@@ -0,0 +1,891 @@
+
+**CAN test in Conductor workspace:**
+
+- ✅ RuboCop linting
+- ✅ RSpec unit tests (gem-only)
+- ✅ Prettier formatting
+- ✅ TypeScript compilation
+- ✅ Jest unit tests
+
+**When blocked by environment, CLEARLY STATE:**
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)